### PR TITLE
issue templates: Add four different types of templates (closes #16)

### LINKED ISSUE #16: Come up with labeling scheme to keep things organized
# Summary

Add more issue labels to better include other types of tasks added to this repository

# Background

We have a few labels already for different UNICEF cohorts and meetings, but there are some tasks in this repo that do not fit into the existing labels. Adding useful labels for things we could quickly highlight to someone in a passing conversation (e.g. issues about resources, talks we've given, etc.) helps us identify our areas of specialization.

It also makes it easier to write articles about some of our work too. :sweat_smile: 

# Details

I haven't thought about this in depth yet. Next step for me is come up with a list of labels, get feedback from others if they are actually useful / serve a purpose, and add them.

# Outcome

* Easier to navigate issues in this repo
* Easier to highlight some of our key focus areas, both internally and externally

### PR DESCRIPTION
**Pull request by justwheel**
This commit reorganizes our issue templates. Instead of a single
template for all new tickets, this splits them off into four possible
ticket templates we might expect someone to use. All of them make use of
the metadata to help us automate some of the project management pieces
better, so this remains a useful planning resource for us.

Closes #16.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
```
Comment (Nolski) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: I'm a bit confused as to what bug would refer to in the context of our tasks repository. If there's a bug on one of our actual software projects, the issue should probably reside there.

I could see the bug analogy being used in the case of tasks as we are doing something wrong. Maybe some simple rephrasing of these would work? I'll suggest some changes for this.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+# Summary
+
+<!-- One sentence description what is wrong. -->
+
+
+# Bug Acceptance Criteria
+
+In order for your bug to be accepted, the following sections must be answered to the best of your ability:
+
+## How to reproduce?
```
Comment (Nolski) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
## What's going wrong?
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+<!-- How would someone else experience the issue you are reporting? -->
+
+
+## Expected behavior
```
Comment (Nolski) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]:
```suggestion
## How do we do it right?
```

**Comment by Nolski** [comment]
@jwflory I added a commit to make the bug template seem a bit more general-purpose and less software focused. Let me know what you think!

**Comment by justwheel** [comment]
Oops, missed this. @Nolski, commit looks good to me. Merging! :ocean:

**Pull request merged**